### ISSUE TITLE
Documentation for using the "Icarus" theme like Rviews??

### ISSUE THREAD

**Issue by znmeb** [rstudio/blogdown#107]
I was looking at the R Views site the other day (<https://rviews.rstudio.com/>). That's pretty much exactly the format I want for my Blogdown blog, so I started looking at it in detail and discovered that it was built with Blogdown using a fork of the Hugo "Icarus" theme (<https://github.com/yihui/hugo-icarus-theme>).

So I set up a blog using the original "digitalcraftsman/hugo-icarus-theme" in Blogdown. I had to do some major violence to the templates to get things the way I wanted, though, and it seems to me it would be easier to use the setup you're using for R Views as a starter. Is there some documentation on how R Views is built / maintained?

The hacked-up site is at <https://www.znmeb.mob>. Note that it doesn't have any `.Rmd` content yet; I could have built it with just Hugo.

**Comment by yihui** [comment]
I don't recommend you to use my fork. It is quite a few commits behind the upstream repo now. For RViews, I ended up with pretty heavy customizations for the theme without considering possible general use of other people, i.e. many things are probably hardcoded. So I don't recommend to use the modified theme, either. Anyway, here is the link to download it: [rviews-theme.zip](https://github.com/rstudio/blogdown/files/988920/rviews-theme.zip)

I don't want to provide support for other themes except the default one (hugo-lithium-theme), because it is extremely time-consuming to learn a theme thoroughly and tweak it. Themes can be wildly different with each other. Personally I'm tired of fancy themes after blogging for about 12 years. The more effort you investigate into a single theme, the more painful it will be to leave it when you are tired of it.

I'll definitely write about some general knowledge of Hugo themes in the book, though.

**Comment by znmeb** [comment]
Ah ... OK. I'm probably sticking with the Icarus theme from Digital Craftsman for both of my sites - it's the only one I found that has everything I need and looks decent.

When I was blogging with Nikola I ended up with a simple workaround for R Markdown documents that I'm probably going to use with Blogdown. I kept a separate folder with the R Markdown documents, rendered them to Nikola's Markdown with knitr and Pandoc and then moved the rendered documents where Nikola expected them before generating the site.
